platforms/nuttx/src/px4/nxp/s32k3xx/board_reset/board_reset.cpp

```diff
@@ -80,15 +80,20 @@ static int board_reset_enter_bootloader()
 
 int board_reset(int status)
 {
-	if (status == 1) {
-		board_reset_enter_bootloader();
-	}
+	board_functional_reset();
```

Comment:
@PetervdPerk-NXP For the FMU's when connected to USB there is a delay in the bootloader that will allow 5 Sec for the host to request bootloading.  From the command line and via QGC there is an  option to reboot and stay in the bootloader. The command line -b is used to communicated to the bootloader to stay in boot load mode. The RTC registers are used with a signature to facilitate this. 

Did you purposely choose not to implement this or is the removal of line 83 an oversite?

Reply:
It's because we simply don't have a bootloader for the S32K3 yet.